common/src/main/resources/Operator.properties

```diff
@@ -201,16 +201,16 @@ WLSDO-0014=The domain {0} introspect job failed, it will automatically retry in
   retries = {2}
 WLSDO-0015="Retrying failed domain {0} introspect job, retry {1} of a maximum {2}."
 WLSDO-0016=DomainUID ''{0}'' introspector job name ''{1}'' exceeds maximum allowed length ''{2}''.
-WLSDO-0017=DomainUID ''{0}'' and cluster name ''{1}'' combination ''{2}'' exceeds maximum allowed length ''{3}''.
```

Comment:
The message is probably clearer with the combination in it because the combination includes additional characters, such as "-", besides the domainUID and cluster name.

Reply:
I spoke with Rosemary, who felt that it made the message actually more difficult to understand.

Reply:
we could reword it to make it easier to understand, but the key is that we need to show the generated name in the message so that customer knows why it exceeds the limit.

Reply:
Why don't we discuss with Rosemary; I think that showing the generated message doesn't provide any more information than the number of total characters allowed.

Reply:
An example may help here. 

Let us say we have a domain with UID mydomain1, and a cluster mycluster1. The generated service name for this cluster is mydomain1-cluster-mycluster1, which has 28 characters. If the limit is 25 characters (for the sake of the discussion), the customer would see an error message like

The combination of UID "mydomain1" and cluster name ''mycluster1" exceeds maximum allowed length 25. 

But the total number of characters in domainUID and cluster name is only 19.

Reply:
What about something like the following?

WLSDO-0017=The generated service name "{3}" for domain ''{0}'' and cluster name ''{1}'' exceeds maximum allowed length ''{2}''.

Reply:
In all cases where the generated name is too long, the number displayed in the error message is the number allowed for the combination of uid and name. So in your case, it would say 

The combination of UID "mydomain1" and cluster name ''mycluster1" exceeds maximum allowed length 16. The number displayed is not the maximum size of the _generated_ name. That would indeed not be reasonable. It is the maximum characters that the user is allowed to specify.

Reply:
I think that showing the user the generated name makes it clearer to the user.

Reply:
@doxiao Please show me where you think that is happening. The unit test explicitly asserts that the maximum length allowed is the length of the user-specified names. 

My objection to including both the user-specified names and the resultant combined name is that the message has a lot of duplicate information and is difficult to read, given that it is close to two hundred chars long, and wrapped several times in all the places we display it.

Reply:
Since your change has deducted the number of additional characters that the operator adds to the generated names. The information in the message is correct. The only issue is usability. I am okay with the current change since you have run it with Rosemary.